src/app/pages/tutorial/tutorial.scss

```diff
@@ -1,20 +1,42 @@
 ion-toolbar {
   --background: transparent;
-  --border-color: transparent;
 }
 
-.slide-title {
-  margin-top: 2.8rem;
+ion-content {
+  --overflow: hidden;
 }
 
 .slide-container {
+	display: grid;
+	grid-template-columns: repeat(4, 100%);
+	grid-template-rows: 1fr;
+
+	height: 100%;
+
+  overflow: scroll;
+	scroll-snap-type: x mandatory;
+}
```

Comment:
It looks like this spacing is caused by some of the lines using tabs and others using spaces; could we get those consistent?

Reply:
Added stylelint to autocorrect this and other style issues based on the framework's config: https://github.com/ionic-team/ionic-conference-app/pull/1790/commits/e6c9d27ac644898bbc0d36433ccbd436cf1535d0